modules/services/attic.nix

```diff
@@ -23,6 +23,18 @@ in
     settings = lib.mkOption {
       type = lib.types.submodule {
         freeformType = settingsFormat.type;
+
+        options = {
+          storage = {
+            type = lib.mkOption {
+              type = lib.types.str;
```

Comment:
`type` should be an `enum` i think. And `path` can be `types.path`

Reply:
Type can be `s3` (or something) as well so we probably shouldn't restrict `path` to `types.path`. But it should probably be `enum` yeah.